From 10159fc0e4363bd88f918335746d2da0be18ec88 Mon Sep 17 00:00:00 2001
From: Andrew Butterfield <andrew.butterfield@scss.tcd.ie>
Date: Fri, 4 Aug 2023 17:10:42 +0100
Subject: [PATCH 08/10] submodule sync

---
 modules/rtems | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules/rtems b/modules/rtems
index 4f7315d4..dd20424e 160000
--- a/modules/rtems
+++ b/modules/rtems
@@ -1 +1 @@
-Subproject commit 4f7315d48ed1bb2a3a619457925df9f08e48e5ea
+Subproject commit dd20424ee2a37aa0da1da061e92349384e75deaa
-- 
2.37.1 (Apple Git-137.1)

